control/dtime.py

```diff
@@ -89,32 +89,4 @@ def sample_system(sysc, Ts, method='zoh', alpha=None):
 
     return sysc.sample(Ts, method, alpha)
 
-
-def c2d(sysc, Ts, method='zoh'):
```

Comment:
This removes the documentation for `c2d` doesn't it? Not what we want, probably. This is clearly written as a wrapper.

Reply:
Hi Ben, this PR makes it so both c2d and sample_system are identical (one is equal to the other) so that they have the same documentation. Which is a shortened version of eg. Transferfunction.sample, which is itself a modified version of scipy.signal.cont2discrete's docs. For both cases, do you think it would make sense to just refer the reader to  scipy.signal.cont2discrete, to avoid the need to continually update documentation?

Reply:
I know, but are you sure that your change retains the documentation for c2d in https://python-control.readthedocs.io/en/0.8.3/generated/control.matlab.c2d.html? The call signature will change for sure, because `c2d()` did not have the `alpha` parameter before.

Reply:
Hi Ben, do you think it's ok if c2d gets an alpha parameter now (that matlab doesn't have?) 

Either way, I can update the code to duplicate its docstring so that the docstring example is for c2d rather than sample_system.